ocs_ci/cleanup/aws/defaults.py

```diff
@@ -0,0 +1,7 @@
+"""
+Constants module for AWS cleanup
+"""
+DEFAULT_AWS_REGION = 'us-east-2'
```

Comment:
This can probably just be `AWS_REGION` now that it's in a defaults module, but not a big deal :)